tooling/template-base-essentials/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@thoughtindustries/helium-template-essentials",
+  "version": "1.2.0-beta.6",
+  "scripts": {
+    "authenticate": "helium authenticate",
+    "deploy": "helium deploy",
+    "dev": "helium dev",
+    "update-translations": "helium update-translations",
+    "dev:vite": "concurrently \"npm run dev:css\" \"npm run dev:server\"",
+    "dev:css": "tailwindcss -o ./renderer/tailwind.css --watch",
+    "dev:server": "tsup ./server/index.ts && node ./dist/index.js",
+    "build": "npm run build:css && npm run build:vite",
+    "build:css": "tailwindcss -o ./renderer/tailwind.css",
+    "build:vite": "vite build",
+    "build:atoms": "vite build -c vite.atoms.config.ts && vite build"
+  },
+  "publishConfig": {
+    "access": "public",
+    "tag": "beta"
+  },
+  "dependencies": {
+    "@apollo/client": "^3.7.0",
+    "@cloudflare/kv-asset-handler": "^0.2.0",
+    "@graphql-tools/graphql-tag-pluck": "^7.1.3",
+    "@heroicons/react": "^2.0.13",
+    "@mdx-js/rollup": "^2.1.3",
+    "@tailwindcss/line-clamp": "^0.3.1",
+    "@thoughtindustries/cart": "^1.2.0-beta.3",
+    "@thoughtindustries/catalog": "^1.2.0-beta.3",
+    "@thoughtindustries/content": "^1.2.0-beta.3",
+    "@thoughtindustries/featured-content": "^1.3.0-beta.3",
+    "@thoughtindustries/helium-server": "^2.3.0",
+    "@thoughtindustries/hero": "^1.1.0-beta.1",
+    "@thoughtindustries/navigation-bar": "^1.2.0-beta.1",
+    "clsx": "^1.2.1",
+    "concurrently": "^7.0.0",
+    "cross-env": "^7.0.3",
+    "graphql": "^16.6.0",
+    "i18next": "^21.10.0",
+    "jwt-decode": "^3.1.2",
+    "node-html-parser": "^4.1.5",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-h5-audio-player": "^3.8.6",
+    "react-i18next": "^11.18.6",
+    "remark-frontmatter": "^4.0.1",
+    "remark-mdx-frontmatter": "^2.0.3",
+    "rollup-plugin-external-globals": "^0.7.1",
```

Comment:
Can remove this dependency from this `package.json` since `atoms` isn't included here -- same for the `template-base` `package.json`.